lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.test.js

```diff
@@ -744,4 +744,59 @@ describe('#updateStage()', () => {
   it('should enable tracing if fullExecutionData is set to true', () => {
     return checkDataTrace(true);
   });
+
+  it('should allow a apiGw stage name that differs from the sls stage', () => {
+    context.state.service.provider.apiGateway = {
```

Comment:
All new tests should be configured with `runServerless` util. See: https://github.com/serverless/serverless/tree/master/test#unit-tests